pages/mesosphere/dcos/services/kubernetes/2.6.0-1.17.7/release-notes/index.md

```diff
@@ -3,7 +3,7 @@ layout: layout.pug
 navigationTitle: Release Notes
 title: Release Notes
 menuWeight: 10
-excerpt: Release notes for DC/OS Kubernetes version 2.6.0-1.17.7
+excerpt: DC/OS Service for Kubernetes version 2.6.0-1.17.7 Release Notes
```

Comment:
@cmastr This is not inconsistent with the previous packages release notes - is this what they should all look like now? Please also remember that we use the source repo (https://github.com/mesosphere/dcos-kubernetes-cluster) as the authoritative source for docs and merge them over here - if you need this changing, I will backport this change to the source repo too (my day yesterday was spent aligning changes here in this docs site with the source repo :disappointed: so I'm mentioning this now.